### PR TITLE
[docs] Release notes for 2.17.3.0

### PR DESCRIPTION
**Pull request by aishwarya24**
Note to self : Tentative Rc build -142. Update as needed.

@netlify /preview/releases/yba-releases/v2.17/

### CODE REVIEW

File: docs/content/preview/releases/yba-releases/v2.17.md [hunk 1]
```diff
@@ -24,6 +24,288 @@ Included here are the release notes for all releases in the **YugabyteDB Anywher
...
+* [PLAT-5310] Add search functionality for universe tables lists
+* [PLAT-5368] [UI] Implement Custom Notification Email Template
+* [PLAT-5370] [UI] Add Custom Variable Support for Email Notification Template
+* [PLAT-5380] Set gFlag to use hostname for onprem TLS enabled universes using DNS
+* [PLAT-5443] Enabling python 3.10 and 3.11 support for ybcloud
+* [PLAT-5712] Account for disks for paused universe
+* [PLAT-5734] Support EAR enabled universes for attach/detach
+* [PLAT-5796] [YBA Enablement] Cloud Provider Config - Onprem(sudo, non-sudo)
+* [PLAT-5888] [PLAT-5889] Adding Re-encryption modules for all KMS providers universe key
+* [PLAT-6559] Add create on prem provider form
```
Comment (ddhodge) [inline docs/content/preview/releases/yba-releases/v2.17.md:51]: Two 6559s?

File: docs/content/preview/releases/yba-releases/v2.17.md [hunk 2]
```diff
@@ -24,6 +24,288 @@ Included here are the release notes for all releases in the **YugabyteDB Anywher
...
+* [PLAT-7682] Bump up prometheus/postgres version
+* [PLAT-7690] Remove deprecated field firstTry from universe details json
+* [PLAT-7691] Change fontweight of MaterialUI YBlabel component
+* [PLAT-7692] Change default AWS instance type to c5.4xlarge
+* [PLAT-7695] Create user with -U to ensure group is created
+* [PLAT-7696] Add --force or --rebuild to build_ansible_pex.sh to prevent accidental deletion of pexEnv
+* [PLAT-7712] Remove wizard flow for create and edit universe
+* [PLAT-7730] File upload UI improvments
+* [PLAT-7730] Use flex-start instead of start
+* [PLAT-7734] Disable delete provider action when provider in use
```
Comment (ddhodge) [inline docs/content/preview/releases/yba-releases/v2.17.md:193]: two 7734s? these look the same

File: docs/content/preview/releases/yba-releases/v2.17.md [hunk 3]
```diff
@@ -24,6 +24,288 @@ Included here are the release notes for all releases in the **YugabyteDB Anywher
...
+* [PLAT-6609] [PLAT-7618] [PLAT-7633] fix Error handling, alignment issues , code cleanup and RR does not inherit tags from primary
+* [PLAT-6859] Fix health check timestamp + universe creation from s3 release.
+* [PLAT-7048] [PLAT-7066] Fixing YBA installer issues with backup/restore
+* [PLAT-7052] [Dedicated Nodes] Add read replica failing for universe with Dedicated Nodes
+* [PLAT-7196] Update options in universe_configure are broken for k8s
+* [PLAT-7303] OCP backup failure due to permission issue when copying over xxhash binary
+* [PLAT-7313] [PLAT-7632] PITR improvements and fixes
+* [PLAT-7359] Fix skip validation submission for AWS create provider
+* [PLAT-7410] Fix deeplink of CPU and Disk Usage to appropriate metrics section
+* [PLAT-7411] [PLAT-7725] - fix for enable/disable volume controls/User cannot change placements
```
Comment (ddhodge) [inline docs/content/preview/releases/yba-releases/v2.17.md:241]: 7411 overlap

File: docs/content/preview/releases/release-notes/v2.17.md [hunk 4]
```diff
@@ -25,6 +25,223 @@ Included here are the release notes for the YugabyteDB v2.17 release series. Con
...
+### Docker
+
+```sh
+docker pull yugabytedb/yugabyte:2.17.3.0-b142
+```
+
+### New Features
+
+* [[7836](https://github.com/yugabyte/yugabyte-db/issues/7836)] [YSQL] Batching IN queries on Hash Keys
+* [[12417](https://github.com/yugabyte/yugabyte-db/issues/12417)] [YSQL] Make PITR code aware of per-database catalog version mode
```
Comment (ddhodge) [inline docs/content/preview/releases/release-notes/v2.17.md:64]: two 12417s?

File: docs/content/preview/releases/release-notes/v2.17.md [hunk 5]
```diff
@@ -25,6 +25,223 @@ Included here are the release notes for the YugabyteDB v2.17 release series. Con
...
+* [[16552](https://github.com/yugabyte/yugabyte-db/issues/16552)] [YSQL] Prevent prefix_length_ from taking non-key columns into account
+* [[16746](https://github.com/yugabyte/yugabyte-db/issues/16746)] xCluster: Compute and propagate apply_safe_time from producer
+* [CLOUD-11878] Allow editing of used provider for YBM
+* [CLOUDGA-12309] Populate providerCode in regions/az even when config is null
+* [DB-5721] [YCQLSH] Updating yugabyte-db-thirdparty release version
+* Adding missing fields to return json from suggested_k8s response.
+* API modifications to allow consumption by Terraform
+* Changing Null Check for pullsecretfile
+* Easy Deployment of K8s cluster, changes to rbac rules.
+* Make diagrams clickable (#16437)
```
Comment (ddhodge) [inline docs/content/preview/releases/release-notes/v2.17.md:189]: remove - docs

File: docs/content/preview/releases/release-notes/v2.17.md [hunk 6]
```diff
@@ -25,6 +25,223 @@ Included here are the release notes for the YugabyteDB v2.17 release series. Con
...
+* Refresh AWS pricing information
+* Set up chrony correctly during AMI build process
+* Show image in popup on click (#16297)
```
Comment (ddhodge) [inline docs/content/preview/releases/release-notes/v2.17.md:192]: same

File: docs/content/preview/releases/release-notes/v2.17.md [hunk 7]
```diff
@@ -25,6 +25,223 @@ Included here are the release notes for the YugabyteDB v2.17 release series. Con
...
+* [[16501](https://github.com/yugabyte/yugabyte-db/issues/16501)] [[14166](https://github.com/yugabyte/yugabyte-db/issues/14166)] [[16520](https://github.com/yugabyte/yugabyte-db/issues/16520)] [DocDB] fix corruption bug in test fixture PgMiniLargeClockSkewTest
+* [[16507](https://github.com/yugabyte/yugabyte-db/issues/16507)] DST PITR + Packed - Fix sys catalog restoration logic
+* [[16763](https://github.com/yugabyte/yugabyte-db/issues/16763)] xCluster: Reduce logging noise during xcluster errors
+* [[16401](https://github.com/yugabyte/yugabyte-db/issues/16401)] [yugabyted] `configure data_placement` not working with constraint value with uppercases.
+* [CLOUDGA-11999] Fix nginx proxy location in charts
+* [CLOUDGA-12305] Missing getters for provider fields
+* [DB-4393] [[15210](https://github.com/yugabyte/yugabyte-db/issues/15210)] YBClient fails to connect if the tserver advertises multiple hosts/ports
+* [DB-5855] xCluster Configuration - Data Type Missing in the Type.Java List
+* Adding missing values to UI for prefill kubeconfig
+* fix entry for dr0pdb (#16352)
```
Comment (ddhodge) [inline docs/content/preview/releases/release-notes/v2.17.md:238]: remove

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/releases/yba-releases/v2.17.md (hunk 1 above)
Comment (aishwarya24) [reply docs/content/preview/releases/yba-releases/v2.17.md:51]: hmm for some reason , delete duplicates haven’t work well this time. Thanks Ill take a closer look.

**Pull request merged**